capstone/src/main/webapp/js/graphs.js

```diff
@@ -0,0 +1,30 @@
+google.charts.load('current', {packages: ['corechart', 'bar']});
+google.charts.setOnLoadCallback(graphGrossNetTuition);
+
+/**
+ * The callback function that will generate the double
+ * bar graph. Uses Google Charts API and JSON object
+ * consisting of college tuition data.
+ */
+function graphGrossNetTuition() {
+  const data = new google.visualization.DataTable();
+  fetch('/financial-data').
+      then((response) => response.json()).
+      then((tuitionInfo) => {
+        // iterate over tuitionInfo JSON
```

Comment:
This comment reads like a TODO. If that is the case then start your comment with `// TODO(github_issue_id): Iterate over tuitionInfo JSON...`